docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -52,35 +52,13 @@ so you need to add peering for internal communication between the Auditor, Ledge
 * You must create peering between Ledger, Auditor, and Client VPCs to enable communication between private networks.
 * You must update Ledger, Auditor, and Client route tables for VPC peering connections.
 
-### Recommendations
-
-* You should create Network ACLs for Ledger and Auditor VPCs.
-* You should set up the Network ACLs of the Ledger and Auditor to restrict unused accesses.
-
 ### Steps
 
 * Create 3 peering connections between the Ledger, Auditor and Client VPCs based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
     * Peering between Ledger and Auditor.
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
```

Comment:
Just removing is fine? Are we adding something in another PR?
Do we have a security group setting at the moment?

Reply:
> Just removing is fine? 

Yes, By default the security group enables ledger and auditor ports only. (I need to check with a new environment, Now I checked with the PSIM environment)

> Are we adding something in another PR?

I will create a new PR for the `kota2and3kan` san's comment.

> Do we have a security group setting at the moment?

No, In my understanding security group enables ledger and auditor ports.